src/artifact/url.js

```diff
@@ -3,56 +3,47 @@
  * @overview artifact urls helper
  */
 
-import {apiUrl} from '../utils/url';
 import {locatorToString} from '../utils/url';
 
 /**
- * @param {TeamcityApiOptions} options
  * @param {Object} locatorObject
  * @param {String} path
  * @returns {String}
  */
-export function contentUrl(options, locatorObject, path) {
-    const baseUrl = apiUrl(options);
+export function contentUrl(locatorObject, path) {
     const locator = locatorToString(locatorObject);
```

Comment:
`locatorObject` is pure `js` object, that specified by client.
For example:

```js
var locator = {
  buildType: {
    id: 123
  }
};

client.build.detail(locator)
  .then(processResponse);
```

I think, it's harmfull to force user import additional class from `teamcity-client` package.

```js
var client = require('teamcity-client');
var Locator = require('teamcity-client/locator');

var locator = new Locator({
  buildType: {
    id: 123
  }
});
client.build.detail(locator)
  .then(processResponse);
```